src/Core/Framework/Adapter/Filesystem/PrefixFilesystem.php

```diff
@@ -22,7 +23,7 @@ public function __construct(
         string $prefix
     ) {
         if (empty($prefix)) {
-            throw new \InvalidArgumentException('The prefix must not be empty.');
```

Comment:
if someone relies on catching an InvalidArgumentException, this is a break. please use a feature flag here